19 redis

jberns opened the pull request:
This closes #19 to cache the calculated data for each county after it is calculated the first time.

An `.env` file will be required to run this branch in order to interact with the redis server.

pjhoberman commented on backend/main.py:
```diff
@@ -19,10 +37,33 @@ async def counties():
-        return {"data": county_data}
+
+        county_cache_exists = r.exists(county)
+        if (county_cache_exists == 1 ): # 1 = exists, 0 = does not exist
```
Might be a little less readable, but do you want to do something like

`if r.exists(county): `

pjhoberman commented on backend/tests/denver_county_raw.py:
```diff
@@ -0,0 +1 @@
+raw = "{\"2010\":{\"Population\":578087,\"Households\":254181,\"Average Household Size\":2.23,\"Median Household Income\":45501,\"Households below 200% FPL\":99021,\"% Households below 200% FPL\":0.39,\"EA Participants\":\"3594\",\"LEAP Participants\":\"15926\",\"Total Households Assisted\":19520,\"% of Households below 200% FPL Assisted\":0.2},\"2011\":{\"Population\":590507,\"Households\":25813 ...
```
What about combining the three denver data files into one, and then importing from the one file in tests?

jberns replied:
Updated this.  I think this makes a lot more sense and is way cleaner.

pjhoberman commented on backend/utils/date_helper.py:
```diff
@@ -0,0 +1,5 @@
+from datetime import datetime
```
What about combining some of the utils files into one. perhaps `helpers.py` or something could include date helper and redis helper

jberns replied:
I recombined these for now.  If it gets too big we can split out function later.

jberns commented:
> Have you seen that?

I think I fixed that testing issue.  I added setup and teardown functions so the redis helper is removed before moving to the next test which fixed this error.
